BangCommands/Helper.js

```diff
@@ -104,6 +104,7 @@ var BangCommandHelper =
 		var foundPath = "";
 
 		var paths = [
+            path.resolve("/.smelt/plugins/" + name + ".js"),
```

Comment:
I dont know why, but with the code that was looking for plugins in the local directory looked for `c://.smelt/plugins/` instead of local when running from smelt-plugins.js. with that addition it seemed to run fine.